pom.xml

```diff
@@ -1215,7 +1215,7 @@
       <dependency>
         <groupId>org.apache.ant</groupId>
         <artifactId>ant</artifactId>
-        <version>1.9.15</version>
+        <version>1.10.9</version>
```

Comment:
This seems okay to me, not sure about the windows build failure

Reply:
windows build passed, so it's good to go AFAICS